examples/third-party-resources/README.md

```diff
@@ -0,0 +1,43 @@
+# Third Party Resources Example
+
+This particular example demonstrates how to perform basic operations such as:
+
+* How to register a new ThirdPartyResource (custom Resource type)
+* How to create/get/list instances of your new Resource type (update/delete/etc work as well but are not demonstrated) 
+
+## Running
+
+```
+# assumes you have a working kubeconfig, not required if operating in-cluster
+go run *.go -kubeconfig=$HOME/.kube/config
+```
+
+## Use Cases
+
+ThirdPartyResources can be used to implement custom Resource types for your Kubernetes cluster.
+These act like most other Resources in Kubernetes, and may be `kubectl apply`'d, etc.
```

Comment:
Do you mean you can `kubectl apply` ThirdpartyResource, or instances of ThirdpartyResource? If it's the former case, I feel it doesn't need to be emphasized.